mediapipe/graphs/instantmotiontracking/calculators/obj3d/BUILD

```diff
@@ -0,0 +1,476 @@
+# Copyright 2020 The MediaPipe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//mediapipe/framework/port:build_config.bzl", "mediapipe_cc_proto_library")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+proto_library(
+    name = "object_proto",
+    srcs = [
+        "object.proto",
+    ],
+)
+
+proto_library(
+    name = "a_r_capture_metadata_proto",
+    srcs = [
+        "a_r_capture_metadata.proto",
+    ],
+)
+
+proto_library(
+    name = "annotation_proto",
+    srcs = [
+        "annotation_data.proto",
+    ],
+    deps = [
+        ":a_r_capture_metadata_proto",
+        ":object_proto",
+    ],
+)
+
+proto_library(
+    name = "belief_decoder_config_proto",
```

Comment:
It looks like most of these are not used by us-- even though we're replacing the other BUILD file, let's just include the rules here that we're using ourselves.